frontend/src/components/FormControl/FormLabel/FormLabel.tsx

```diff
@@ -52,6 +52,11 @@ export interface FormLabelProps extends ChakraFormLabelProps {
    * Whether markdown is enabled for description text.
    */
   useMarkdownForDescription?: boolean
+
```

Comment:
same - to include the why instead of what which is self explanatory. I understand that you're following the previous comment style, but since we are trying to improve the code quality over time, including why instead of what is more beneficial.

Reply:
source: https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/